Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -1272,6 +1295,226 @@ def fetch_incidents(client: Client,
+            args['incident_changed']: boolean telling us if the local incident indeed changed or not
+            args['remote_incident_id']: the remote incident id
+        params:
+            entry_tags: the tags to pass to the entries (to separate between comments and work_notes)
+
+    Returns: The remote incident id - ticket_id
+
+    """
+
+    parsed_args = UpdateRemoteSystemArgs(args)
+    if parsed_args.delta:
+        demisto.debug(f'Got the following delta keys {str(list(parsed_args.delta.keys()))}')
+
+    # ticket_type = client.ticket_type
+    ticket_id = parsed_args.remote_incident_id
+    if parsed_args.incident_changed:
+        demisto.debug(f'Incident changed: {parsed_args.incident_changed}')
+        # Close ticket if needed
+        if parsed_args.inc_status == IncidentStatus.DONE and params.get('close_ticket'):
+            # Set status TOPdesk ticket to Closed
+            demisto.debug('Close TOPdesk ticket')
+            # Close with API call or set field and let mirroring handle it.
+            # client.update_incident
+        # TODO: Something with updated delta keys or not?
+        # 'processingStatus', 'priority', 'urgency', 'impact'
+        update_args = {
+            'id': ticket_id
+        }
+        for key in parsed_args.delta:
+            update_args[key] = parsed_args.delta[key]
+
+        demisto.debug(f'SZU update_args=[{update_args}]')
+        client.update_incident(update_args)
+
+    entries = parsed_args.entries
+    if entries:
+        demisto.debug(f'New entries {entries}')
+
+        for entry in entries:
+            demisto.debug(f'Sending entry {entry.get("id")}, type: {entry.get("type")}')
+            # Mirroring files as entries
+            if entry.get('type') == 3:
+                path_res = demisto.getFilePath(entry.get('id'))
+                full_file_name = path_res.get('name')
+                file_name, file_extension = os.path.splitext(full_file_name)
+                if not file_extension:
+                    file_extension = ''
+                client.attachment_upload(incident_id=ticket_id, incident_number=None, file_entry=entry.get('id'),
+                                         file_name=file_name + '_mirrored_from_xsoar' + file_extension,
+                                         invisible_for_caller=False,
+                                         file_description=f"Upload from xsoar: {file_name}.{file_extension}")
+            else:
+                # Mirroring comment and work notes as entries
+                xargs = {
+                    'id': '',
+                    'action': '',
+                    'action_invisible_for_caller': False,
+                }
+                tags = entry.get('tags', [])
+                if params.get('work_notes_tag') in tags:
+                    xargs['action_invisible_for_caller'] = True
+                elif params.get('comments_tag') in tags:
+                    xargs['action_invisible_for_caller'] = False
```

Comment:
This can be removed as the default value for `action_invisible_for_caller` is False.